svgnative/src/SVGStringParser.cpp

```diff
@@ -403,8 +412,14 @@ void ParsePathString(const std::string& pathString, Path& p)
             return;
         if (!isDigit(*pos) && *pos != ',' && *pos != '-' && *pos != '.')
             prev = *pos++;
-        else if (prev == 'z' || prev == 'Z')
+        else if (IsCloseCmd(prev))
             break;
+        // Do not close a path if we do not have one yet.
+        if (IsCloseCmd(lastCommand) && IsCloseCmd(prev))
+            continue;
```

Comment:
So prev is ignored in this case? Shouldn't we warn about this?

Reply:
In SVG it is valid to have multiple close paths commands w/o creating a new subpaths. This change makes sure we don't call closePath on the renderer multiple times.